src/index.css

```diff
@@ -1 +1,5 @@
 @import 'tailwindcss';
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
```

Comment:
There is a better way to setup fonts in tailwind theme instead of using native css
check here https://tailwindcss.com/docs/font-family#customizing-your-theme